### ISSUE TITLE
Testing on custom video

### ISSUE THREAD

**Issue by javierlorenzod** [facebookresearch/SlowFast#152]
Hello everyone,
First of all, thanks for sharing your code. 
I was wondering if there is a way of testing an AVA trained detection model on a video from internet without having to follow the structure of the dataset. In #137 you talked about the demo.gif and I want  to know if it can be reproduced.

Thank you.

**Comment by sdjsngs** [comment]
Hi have you solve this problem?   thank you

**Comment by javierlorenzod** [comment]
No, sorry, currently I do not have time to do it. I plan to do it in the near future. When I complete it, I will make my fork public for everyone.
